### PR TITLE
[2.21.x] Update policies and interceptors

### PR DESCRIPTION
**Pull request by aaronilovici**
#### What does this PR do?
This PR removes the requirement for a guest user to be able to do CRUD operations, as well as changes to the PEPAuthInterceptor that allow for SessionToken creation for requests that do not have an authentication token.

#### Who is reviewing it? 
@rfding 
@mdang8 
@josephthweatt 

#### Select relevant component teams: 
@codice/security 

#### Ask 2 committers to review/merge the PR and tag them here.
@clockard
@stustison

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: catalog/security/catalog-security-policyplugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml [hunk 1]
```diff
@@ -18,22 +18,22 @@
                                update-strategy="container-managed" />
         <property name="createPermissions">
             <array>
-                <value>http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=guest</value>
+                <value>""</value>
```
Comment (clockard) [inline catalog/security/catalog-security-policyplugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml]: I haven't looked at this in a while but does this mean that by default no one will be able create/update/delete?

Comment (stustison) [reply catalog/security/catalog-security-policyplugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml]: It means exactly the opposite. Also, I don't think the "" is what we necessarily want here. I think we can just leave these properties completely unset.

Comment (aaronilovici) [reply catalog/security/catalog-security-policyplugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml]: Nope, this means there are no restrictions as to who can perform CRUD operations.

Comment (aaronilovici) [reply catalog/security/catalog-security-policyplugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml]: Ah Scott beat me to it. And also okay, I can just remove the string altogether.

**Comment by aaronilovici** [comment]
build now

File: catalog/security/catalog-security-policyplugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml (hunk 1 above)
Comment (lessarderic) [reply catalog/security/catalog-security-policyplugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml]: Should we add a comment to clarify that?

Comment (aaronilovici) [reply catalog/security/catalog-security-policyplugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml]: Added comments to the blueprints to explain the empty values.

**Comment by aaronilovici** [comment]
build now

**Comment by aaronilovici** [comment]
build now

**Comment by aaronilovici** [comment]
build now

**Pull request merged**